optax/_src/alias.py

```diff
@@ -142,10 +141,13 @@ def rmsprop(learning_rate: ScalarOrSchedule,
 def sgd(learning_rate: ScalarOrSchedule,
         momentum: float = 0.,
```

Comment:
Lets have `None` as default,
and check `if momentum is not None`
instead of `if momentum > 0.`

this works better if for instance you want to decay the `momentum` to zero over time,

Reply:
Done!